Description: Pillow not required
 README.rst says PIL 1.1.7 is fine.
Author: Brian May <bam@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2013-05-29

--- django-simple-captcha-0.3.7.orig/setup.py
+++ django-simple-captcha-0.3.7/setup.py
@@ -25,6 +25,5 @@ setup(
         'setuptools',
         'six >=1.2.0',
         'Django >= 1.3',
-        'Pillow >=2.0.0'
     ]
 )
